src/components/Image/ImageBanner/ImageBanner.scss

```
@@ -9,13 +9,13 @@
     width: 100%;
   }
 }
-@media screen and (max-width: $breakpoint-mobile){
-  .image-banner{
+@media screen and (max-width: $breakpoint-mobile) {
+  .image-banner {
      grid-template-columns: 1fr;
   }
 
-  .image-banner{
-    picture:not(:first-child){
+  .image-banner {
+    picture:not(:first-child) {
```

cant we just chain these? 
```suggestion
    picture:not:first-child {
```

---

[needs the parenthesis ](https://developer.mozilla.org/en-US/docs/Web/CSS/:not)

```
 .image-banner {
     grid-template-columns: 1fr;

     picture:not(:first-child) {
      display: none;
    }
  }
```